backend/pkg/api/data_access/vdb_blocks.go

```diff
@@ -267,12 +266,17 @@ func (d *DataAccessService) GetValidatorDashboardBlocks(ctx context.Context, das
 					goqu.C("slot"),
 					groupId,
 					goqu.V("0").As("status"),
-					goqu.V(nil).As("exec_block_number"),
-					goqu.V(nil).As("graffiti_text"),
-					goqu.V(nil).As("fee_recipient"),
-					goqu.V(nil).As("el_reward"),
+					goqu.L("NULL::INTEGER").As("exec_block_number"),
+					goqu.L("NULL::TEXT").As("graffiti_text"),
+					goqu.L("NULL::BYTEA").As("fee_recipient"),
+					goqu.L("NULL::NUMERIC").As("el_reward"),
```

Comment:
This got quite complicated here.
As soon as `scheduledDs` gets a `LIMIT` added to it the `Union` goqu method determines that it now needs to wrap itself, meaning `SELECT ...` => `SELECT * FROM ( SELECT ... LIMIT ...) AS t1`
As a result of that it could no longer correctly identify the types of our NULL values.
The only solution I found is adding such casts.